src/db/seeders/20240415171636-user.ts

```diff
@@ -0,0 +1,30 @@
+import { QueryInterface } from 'sequelize';
+
+module.exports = {
+  async up(queryInterface: QueryInterface) {
+    await queryInterface.bulkInsert('users', [
+      {
+        id: 1,
+        name: 'John Doe',
+        email: 'John@gmail.com',
+        roleId: 1,
+        address: 1,
+        createdAt: new Date(),
+        updatedAt: new Date(),
```

Comment:
No need to send user table with dummy values maybe set something like an admin account